packages/@storefront/flux-capacitor/test/unit/core/actions/validators.ts

```diff
@@ -237,6 +257,27 @@ suite('validators', ({ expect, spy, stub }) => {
     });
   });
 
+  describe('areMultiplePastPurchaseRefinementsDeselectedByIndex', () => {
+    const navigationId = 'color';
+
+    it('should be valid if some refinements are deselected', () => {
+      const state: any = { a: 'b' };
+      stub(Selectors, 'pastPurchaseNavigation').withArgs(state, navigationId).returns({ selected: [4, 5, 6, 7] });
+
+      // tslint:disable-next-line max-line-length
+      expect(validators.areMultiplePastPurchaseRefinementsDeselectedByIndex.func({ navigationId, indices: [1, 2, 3, 4, 5, 6, 7] }, state)).to.be.true;
+    });
+
+    it('should be invalid if all refinements are selected', () => {
+      const state: any = { a: 'b' };
+      // tslint:disable-next-line max-line-length
+      stub(Selectors, 'pastPurchaseNavigation').withArgs(state, navigationId).returns({ selected: [1, 2, 3, 4, 5, 6, 7, 8] });
```

Comment:
This array should be pulled out into a `const` and spread here.

Reply:
Done, resolving conversation.